### DISCUSSION TITLE
Mirth Launcher connecting with different port number

### DISCUSSION THREAD

**Discussion by muhcdan** [nextgenhealthcare/connect#5911] [Q&A] [answered]
Hi I noticed that when I launch mirth launcher to a dockerized version using a different port number it connects but the dialog which asks me to login still shows 8443 and I have to change it manually.

Is that a bug? 

TIA

**Comment by jonbartels** [comment, marked as answer]
I think this issue describes your problem - https://github.com/nextgenhealthcare/connect/issues/4655 

There is a community-driven, open-source alternative to the Launcher. You might try https://github.com/kayyagari/ballista and see if it has the same issue or not.
